src/app/styles/base/_variables.scss

```diff
@@ -0,0 +1 @@
+$error-message-color: #ED4337;
```

Comment:
what's the difference between this file & `src/_variables.scss` & `src/app/styles/components/_variables.scss`?